cylc/flow/scripts/lint.py

```diff
@@ -653,12 +653,14 @@ def list_wrapper(line: str, check: Callable) -> Optional[Dict[str, str]]:
         FUNCTION: check_wallclock_directives,
     },
     'S015': {
-        'short': 'Task outputs {outputs}: {description}.',
-        FUNCTION: check_skip_mode_outputs
+        'short': (
+            '``=>`` implies line continuation without ``\\``.'
+        ),
+        FUNCTION: re.compile(r'=>\s*\\').findall
     },
     'S016': {
         'short': 'Run mode is not live: This task will only appear to run.',
-        FUNCTION: re.compile(r'run mode\s*=\s*[^l][^i][^v][^e]$').findall
+        FUNCTION: check_skip_mode_outputs
     },
```

Comment:
It looks like you have accidentally merged the 2 lint checks added in this branch into 1

Reply:
Yep.